UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -197,6 +198,33 @@ namespace UnitsNet
             return new $quantityName($func);
         }
 
+        /// <summary>
```

Comment:
The build server complains about overloads not supported in Windows Runtime Component.

> [C:\projects\unitsnet\UnitsNet\UnitsNet.WindowsRuntimeComponent.csproj]
C:\projects\unitsnet\UnitsNet\GeneratedCode\Quantities\Information.g.cs(67,33,67,44): error WME1085: The 1-parameter overloads of UnitsNet.Information.FromTerabits must have exactly one method specified as the default overload by decorating it with Windows.Foundation.Metadata.DefaultOverloadAttribute. 

Try adding `Windows.Foundation.Metadata.DefaultOverloadAttribute` to the `double?` methods to tell it which overload to use for Windows UWP.
Alternatively wrap all the other overloads in `#if !WINDOWS_UWP ... #endif`, but I think the attribute should do the trick.

Reply:
I'm trying to add the trick. First time using UWP, I've got rid of the build errors locally at least.